packages/callouts/src/sass/_globals.scss

```diff
@@ -0,0 +1,29 @@
+/*! [replace-name] v[replace-version] */
+
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// callouts module globals
+// Use callouts to notify and alert users of important snippets of information.
+//
+// Content:
+// - Sass versioning
+// - Global variables: none so far
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// SASS VERSIONING
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+$name: "[replace-name]";
+$version: "[replace-version]";
+$dependencies: (
+	[replace-dependencies]
+);
+
+@include versioning-add( $name, $version, $dependencies ); //adding dependencies to global scope
+
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// GLOBAL VARIABLES
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+// Boy it sure is quiet here.
+
+$uikit-callout-box-shadow-colour: transparentize($uikit-colour-Text, 0.6);
```

Comment:
Probably — I’ll move it.

Idea had here was to get an rgba value, so that it’s more robust wrt. backgrounds it might be positioned over that aren’t white/we can’t anticipate.